Frontend/ui-driver/src/Screens/ReferralScreen/Controller.purs

```diff
@@ -184,16 +185,18 @@ eval (UpdateLeaderBoard (LeaderBoardRes leaderBoardRes)) state = do
       lastUpdatedAt = convertUTCtoISC (fromMaybe (getCurrentUTC "") leaderBoardRes.lastUpdatedAt) "h:mm A"
   let newState = state{ props { rankersData = rankersData, currentDriverData = currentDriverData, showShimmer = false, noData = not (dataLength > 0), lastUpdatedAt = lastUpdatedAt } }
   _ <- pure $ setRefreshing (getNewIDWithTag "ReferralRefreshView") false
-  if (any (_ == "") [state.props.selectedDay.utcDate, state.props.selectedWeek.utcStartDate, state.props.selectedWeek.utcEndDate]) then do
+  if (any (_ == "") [state.props.selectedDay.utcDate, state.props.selectedWeek.utcStartDate, state.props.selectedWeek.utcEndDate, state.props.selectedMonth.utcDate]) then do
```

Comment:
just we can call Data.String null 
`any DS.null [state.props.selectedDay.utcDate, state.props.selectedWeek.utcStartDate, state.props.selectedWeek.utcEndDate, state.props.selectedMonth.utcDate])`